pkg/apb/client.go

```diff
@@ -175,20 +147,89 @@ func (c *Client) RunImage(
 	c.log.Debug("action:[ %s ]", action)
 	c.log.Debug("extra-vars:[ %s ]", extraVars)
 
+	// TODO(rhallisey): Cleanup the local vs incluster code paths so there
+	// is almost no variation between the two.
+	//
+	// If we're using a locally running broker we want to use the default
+	// namespace.
+	ns := context.Namespace
```

Comment:
@rthallisey @jwmatthews SGTM, will keep 155 since it's just an alias, check for missing namespace (log + error out if missing) and pull 195 ns := "default".